### DISCUSSION TITLE
M2M display template renders as --

### DISCUSSION THREAD

**Discussion by jakobpevec** [directus/directus#16354] [Archive]
### Describe the Bug

Hello,

I have three collections with M2M relations between eachother:
product_variations -> M2M junction table -> product_attribute_values
product_attribute_labels -> M2M junction table -> product_attribute_values

I want the user to select attribute values from a product variation. I also want the user to see (item view) for which attribute label he's picked the value. In the item view (both table and list view) the relation to label will render as --. Does the relation go too deep for directus to pick it up?

![image](https://user-images.githubusercontent.com/69215693/200502208-6a0bedb7-3a10-481d-a174-4f3cc0b5cbb4.png)
![image](https://user-images.githubusercontent.com/69215693/200502342-6482dc1d-7a45-4371-8c5d-8a02f805bf26.png)
https://user-images.githubusercontent.com/69215693/200503851-54f6b940-c214-4fe8-9337-88c0a232b430.mp4

### To Reproduce

Create collections product_variations, product_attribute_values, product_attribute_labels 

Give each collection an input field "Name".

Create a M2M relation between product_attribute_values and product_attribute_labels
Create a M2M relation between product_variations and product_attribute_values

In collection product_variations set the display template interface for M2M field to show the name from product_attribute_labels, across the values table.

### Errors Shown

Name renders as --

### What version of Directus are you using?

9.20.4

### What version of Node.js are you using?

16.18.0

### What database are you using?

10.4.20-MariaDB-1:10.4.20+maria~buster-log

### What browser are you using?

Edge, Firefox, Chrome

### How are you deploying Directus?

Cloudways server, CLI installation, PM2

**Comment by br41nslug** [comment]
This happens when using a nested relational field in a display template is selected which returns more than one value. There currently is no mechanism for the display template to resolve these multiple results and falls back to `--`.

**Reply by jakobpevec** [reply]
Thank you for the explaination. Although if I understand you correctly, my field "Internal name" returns a single value (string input), shouldn't this then display the value?

Is there perhaps a workaround?
